scripts/util/keystore-cert-import

```diff
@@ -121,6 +121,10 @@ if [ -f "$LIBVIRTD_FILE" ]; then
     ln -sf /etc/pki/libvirt/servercert.pem /etc/pki/libvirt-vnc/server-cert.pem
     ln -sf /etc/pki/libvirt/private/serverkey.pem /etc/pki/libvirt-vnc/server-key.pem
     cloudstack-setup-agent -s > /dev/null
+
+    QEMU_GROUP=$(sed -n 's/^group=//p' /etc/libvirt/qemu.conf | awk -F'"' '{print $2}')
```

Comment:
This isn't fullproof and covers the case generally.  Perhaps the logic can be (a) assuming the libvirt group/user can access this file, then this would work; othewise on error detect the distro and (b) use the group "libvirt-qemu" if it's ubuntu, otherwise use the group "qemu" if it's RHEL or one of the other EL clones (I haven't tested for suse15).

As normal non-root user I couldn't run this, but can run as sudoer:

1. On RHEL9:
```
[rohit@plex ~]$ sudo sed -n 's/^group=//p' /etc/libvirt/qemu.conf | awk -F'"' '{print $2}'
root
[rohit@plex ~]$ sed -n 's/^group=//p' /etc/libvirt/qemu.conf | awk -F'"' '{print $2}'
sed: can't read /etc/libvirt/qemu.conf: Permission denied
```

2. On Ubuntu 22.04:
```
> sed -n 's/^group=//p' /etc/libvirt/qemu.conf | awk -F'"' '{print $2}'
sed: can't read /etc/libvirt/qemu.conf: Permission denied
> sudo sed -n 's/^group=//p' /etc/libvirt/qemu.conf | awk -F'"' '{print $2}'
root
```

Reply:
Hi @rohityadavcloud I think this won't be a problem since we just want to access the qemu.conf file from the CloudStack script to obtain the group that the administrator has set, that won't be running as the non-root user. In case qemu doesn't start, then the admin may need to fix the qemu.conf file accordingly